ballerina/tests/mock_service.bal

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/time;
+import ballerina/uuid;
+
+service /api/v1 on new http:Listener(9000) {
+    resource function get account() returns AccountResponse {
+        return {
+            "id": "acc_12345",
+            "trust": {
+                "direct_trust": true,
+                "external_trust": false,
+                "trustAll": false
```

Comment:
use field names instead of keys. Check similar places.

```suggestion
            id: "acc_12345",
            trust: {
                direct_trust: true,
                external_trust: false,
                trustAll: false
```